### PR TITLE
animdl: add page

### LINKED ISSUE #13854: Page request: animdl
### Command description

animdl is a highly efficient, powerful and fast anime scraper.

### Command details

_No response_

### Documentation

https://github.com/justfoolingaround/animdl

### Platform

Common

### VCS repository link (e.g. GitHub, GitLab)

https://github.com/justfoolingaround/animdl

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Arnav-2004**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.7.27

Closes #13854

### CODE REVIEW

File: pages/common/animdl.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`animdl search {{anime_title}}`
+
+- Stream your favorite anime:
+
+`animdl stream {{anime_title}}`
+
+- Stream the latest episode of your favorite anime:
+
+`animdl stream {{anime_title}} -s latest`
```
Comment (Managor) [inline pages/common/animdl.md:36]:
```suggestion
`animdl stream {{anime_title}} {{-s|--special}} latest`
```

File: pages/common/animdl.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Download a specific anime by specifying download directory:
+
+`animdl download {{anime_title}} {{-d|--download-dir}} {{path/to/download_directory}}`
+
+- Grab the stream URL for a specific anime:
+
+`animdl grab {{anime_title}}`
+
+- Display the anime schedule:
```
Comment (Managor) [inline pages/common/animdl.md:22]: For some reason when I try to build this, it fails so I can't test it. Can you provide examples on what this outputs? I feel like the wording there feels clunky and adding more context would make it flow better. Like for example `Display the current season anime schedule`

Comment (Arnav-2004) [reply pages/common/animdl.md:22]: Could you please let me know how to build it locally?

Comment (Managor) [reply pages/common/animdl.md:22]: I tried to use `pip install animdl` in a ven, I tried to use the AUR package and I tried to run 
```
git clone https://www.github.com/justfoolingaround/animdl.git \
&& cd animdl \
&& poetry build \
&& pip install animdl \
&& cd ..
```
as instructed on their github page but all of them got hitched on the same spot

Comment (Arnav-2004) [reply pages/common/animdl.md:22]: Oh you meant `animdl` I thought `tldr`.

Result for `animdl schedule`:
image

Comment (Managor) [reply pages/common/animdl.md:22]: That's super weird. Why is it in reverse order?

Comment (Managor) [reply pages/common/animdl.md:22]:
```suggestion
- Display upcoming anime schedule for the next week:
```

Comment (Arnav-2004) [reply pages/common/animdl.md:22]: > That's super weird. Why is it in reverse order?

No idea, that's how they do it

**Review by Managor — APPROVED** [review]
That looks good to me. Thank you.

File: pages/common/animdl.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# animdl
+
+> A highly efficient, powerful and fast anime scraper.
+> More information: <https://github.com/justfoolingaround/animdl>.
+
+- Download a specific anime:
+
+`animdl download {{anime_title}}`
+
+- Download a specific anime by specifying episode range:
```
Comment (gutjuri) [inline pages/common/animdl.md:10]:
```suggestion
- Download a specific anime by specifying an episode range:
```

File: pages/common/animdl.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`animdl download {{anime_title}} {{-r|--range}} {{start_episode}}-{{end_episode}}`
+
+- Download a specific anime by specifying download directory:
```
Comment (gutjuri) [inline pages/common/animdl.md:14]:
```suggestion
- Download a specific anime by specifying a download directory:
```

File: pages/common/animdl.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`animdl download {{anime_title}} {{-d|--download-dir}} {{path/to/download_directory}}`
+
+- Grab the stream URL for a specific anime:
+
+`animdl grab {{anime_title}}`
+
+- Display upcoming anime schedule for the next week:
```
Comment (gutjuri) [inline pages/common/animdl.md:22]:
```suggestion
- Display the upcoming anime schedule for the next week:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some suggestions

File: pages/common/animdl.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# animdl
+
+> A highly efficient, powerful and fast anime scraper.
```
Comment (spageektti) [inline pages/common/animdl.md:3]:
```suggestion
> A highly efficient, powerful and fast anime scraper.
> See also: `ani-cli`.
```
Similar tool.

**Review by spageektti — COMMENTED** [review]
Hi. Thanks for your contribution.

**Review by spageektti — APPROVED** [review]

**Pull request merged**